docs/developer/stock-tracking.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Stock Tracking
+---
+
+## Tracking Stock in Saleor
+
+In Saleor, inventory management is handled at the product variant level. You can assign specific stock levels to each product variant and decide whether Saleor should track inventory for these variants. This is controlled by the [`productVariant.trackInventory`](api-reference/products/objects/product-variant.mdx#productvarianttrackinventoryboolean---) field. If you want to set a default behavior for all product variants, you can adjust the [`trackInventoryByDefault`](api-reference/miscellaneous/objects/shop.mdx#shoptrackinventorybydefaultboolean--) field on the Shop object.
+
+The warehouse from which stock is reduced is determined by the [Allocation Strategy](developer/stock-allocation.mdx#allocation-strategies).
```

Comment:
As a reader I would like to know what the default behavior, a link or summary would be helpful